components/FormFlashcard/FormFlashcard.js

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+
+export default function FormFlashcard({ collections, onCreateFlashcard }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const newFlashcard = Object.fromEntries(formData);
+
+    onCreateFlashcard(newFlashcard);
+    event.target.reset();
+  }
```

Comment:
**Optional:** Consider to focus the first input element after the form is reset. This can improve the user experience by allowing the user to start typing a new flashcard immediately without needing to click on the first input field again.


```suggestion
  function handleSubmit(event) {
    event.preventDefault();

    const formData = new FormData(event.target);
    const newFlashcard = Object.fromEntries(formData);

    onCreateFlashcard(newFlashcard);
    event.target.reset();
   event.target.elements.question.focus();
  }
```

Reply:
We didn't do this on purpose because currently the form is on the same page as the list of flashcards and we thought maybe the user wants to start learning instead of creating more flashcards. When we separate the form from the list this will make sense. Thank you!